src/lsst/cm/tools/core/error_code_decisions.yaml

```diff
@@ -241,6 +305,17 @@ pandaErrorCode:
             rescue: 0
             panda: 0
             intensity: 0.001
+        p_remote_io:
+            diagMessage: >
+                sqlalchemy.exc.InternalError: (psycopg2.errors.InternalError_) could not read block .* in file ".*":
+                Remote I/O error
```

Comment:
This was related to the USDF databases corruption in the week of Jan 23rd.

Reply:
We should find a way to mark whether something can be fixed by re-running the whole workflow.